pkg/visitors/find.go

```diff
@@ -94,6 +94,16 @@ func FindFileGUIDPredicate(r guid.GUID) func(f uefi.Firmware) bool {
 	}
 }
 
+// FindFileTypePredicate is a generic predicate for searching file types only.
+func FindFileTypePredicate(t uefi.FVFileType) func(f uefi.Firmware) bool {
```

Comment:
We want to remove more than one type at a time. It would be nice to instead specify entire firmware volumes to run the cleaner on. You don't have to change this for now. I'll file an issue